### PR TITLE
Fix maxDestAmt functionality, network and withdrawable tests

### PR DESCRIPTION
**Pull request by Anyhowclick**

### CODE REVIEW

File: test/sol5/kyberNetwork.js [hunk 1]
```diff
@@ -184,16 +182,78 @@ contract('KyberNetwork', function(accounts) {
...
-            //TODO: RemoveReserveFromNetwork
-            //TODO: ListReservePairs
-            //TODO: DAOContractSet 
-            //TODO: KyberNetworkParamsSet
-            //TODO: KyberNetworkSetEnable
-            //TODO: KyberProxyAdded
-            //TODO: KyberProxyRemoved
-            //TODO: HandlePlatformFee
-            //TODO: KyberTrade
+
```
Comment (ilanDoron) [inline test/sol5/kyberNetwork.js]: can we add describe for this
like reserve listing / removing
So its easier to test only this. or remove only this part.

File: test/sol5/kyberNetwork.js [hunk 2]
```diff
@@ -184,16 +182,78 @@ contract('KyberNetwork', function(accounts) {
...
-            //TODO: DAOContractSet 
-            //TODO: KyberNetworkParamsSet
-            //TODO: KyberNetworkSetEnable
-            //TODO: KyberProxyAdded
-            //TODO: KyberProxyRemoved
-            //TODO: HandlePlatformFee
-            //TODO: KyberTrade
+        
+        it("Remove reserve", async() => {
+            txResult = await tempNetwork.removeReserve(mockReserve.address, 0, {from: operator});
```
Comment (ilanDoron) [inline test/sol5/kyberNetwork.js]: let / const

File: test/sol5/kyberNetwork.js [hunk 3]
```diff
@@ -328,6 +385,29 @@ contract('KyberNetwork', function(accounts) {
                     Helper.assertEqual(actualResult.rateAfterAllFees, zeroBN, "rate with all fees not 0");
                 }
             });
+
+            it("expected rate (different hint types) should be zero with zero srcQty, and all reserves return zero rate", async() => {
+                srcQty = zeroBN;
+                for (hintType of tradeTypesArray) {
```
Comment (ilanDoron) [inline test/sol5/kyberNetwork.js]: isn't this actually many tests?

File: test/sol5/kyberNetwork.js (hunk 1 above)
Comment (Anyhowclick) [reply test/sol5/kyberNetwork.js]: Added it after testing contract events

File: test/sol5/kyberNetwork.js (hunk 2 above)
Comment (Anyhowclick) [reply test/sol5/kyberNetwork.js]: Done....

File: test/sol5/kyberNetwork.js [hunk 4]
```diff
@@ -184,18 +180,82 @@ contract('KyberNetwork', function(accounts) {
...
+
+        it("Set enable", async() => {
+            let txResult = await tempNetwork.setEnable(true, {from: admin});
+            expectEvent(txResult, 'KyberNetworkSetEnable', {
+                isEnabled: true
+            });
+        });
     });
 
+    describe("test adding and removing reserves", async() => {})
```
Comment (ilanDoron) [inline test/sol5/kyberNetwork.js]: but its empty here....

File: test/sol5/kyberNetwork.js [hunk 5]
```diff
@@ -61,7 +57,7 @@ let numReserves;
 let info;
 let hint;
 const tradeTypesArray = [EMPTY_HINTTYPE, MASK_IN_HINTTYPE, MASK_OUT_HINTTYPE, SPLIT_HINTTYPE];
-const tradeStr = ["MASK IN", "MASK OUT", "SPLIT", "NONE"];
+const tradeStr = ["MASK IN", "MASK OUT", "SPLIT", "NO HINT"];
```
Comment (ilanDoron) [inline test/sol5/kyberNetwork.js]: shouldn't trade type and string be aligned?

File: test/sol5/kyberNetwork.js (hunk 4 above)
Comment (Anyhowclick) [reply test/sol5/kyberNetwork.js]: LOL yes, being written now

**Pull request merged**